java/ql/src/Telemetry/AutomodelEndpointCharacteristics.qll

```diff
@@ -0,0 +1,325 @@
+/**
+ * For internal use only.
+ */
+
+private import java
+private import semmle.code.java.dataflow.DataFlow
+private import semmle.code.java.dataflow.TaintTracking
+private import semmle.code.java.security.PathCreation
+private import semmle.code.java.dataflow.ExternalFlow as ExternalFlow
+private import semmle.code.java.dataflow.internal.FlowSummaryImpl as FlowSummaryImpl
+private import semmle.code.java.security.ExternalAPIs as ExternalAPIs
+private import semmle.code.java.Expr as Expr
+private import semmle.code.java.security.QueryInjection
+private import semmle.code.java.security.RequestForgery
+private import semmle.code.java.dataflow.internal.ModelExclusions as ModelExclusions
+import AutomodelSharedCharacteristics as SharedCharacteristics
+import AutomodelEndpointTypes as AutomodelEndpointTypes
+
+module CandidatesImpl implements SharedCharacteristics::CandidateSig {
```

Comment:
Shall we name this something like `FrameworkCandidatesImpl`?

Reply:
yes, and I'll rename the file to `AutomodelFrameworkCharacteristics.qll` as well! 👍